app/components/Form/ProjectMilestoneReportForm.tsx

```diff
@@ -233,6 +236,12 @@ const ProjectMilestoneReportForm: React.FC<Props> = (props) => {
         </Button>
 
         {sortedMilestoneReports.map((node, index) => {
+          schema.dependencies.hasExpenses.oneOf[1].properties.calculatedGrossAmount.default =
+            Number(node.calculatedGrossAmountThisMilestone);
+          schema.dependencies.hasExpenses.oneOf[1].properties.calculatedNetAmount.default =
+            Number(node.calculatedNetAmountThisMilestone);
+          schema.dependencies.hasExpenses.oneOf[1].properties.calculatedHoldbackAmount.default =
+            Number(node.calculatedHoldbackAmountThisMilestone);
```

Comment:
Instead of modifying the schema (the default value is meant to be static) we should set these values in the form data, if it's not already set